spec/services/waste_exemptions_engine/confirmation_pdf_generator_service_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+module WasteExemptionsEngine
+  RSpec.describe ConfirmationPdfGeneratorService do
+    describe ".send_confirmation_email" do
+      it "generates and return a string containing PDF content with the confirmation information" do
+        travel_to Time.local(2018, 1, 1, 1, 5, 0)
```

Comment:
For now this needs to be changed to use `Timecop` e.g. `Timecop.freeze(Time.local(2018, 1, 1, 1, 5, 0))`.

I expand on this in a later comment. In case you need it https://github.com/travisjeffery/timecop#usage